internal/server/instance/drivers/driver_qemu.go

```diff
@@ -114,40 +114,60 @@ const qemuBlockDevIDPrefix = "incus_"
 // qemuMigrationNBDExportName is the name of the disk device export by the migration NBD server.
 const qemuMigrationNBDExportName = "incus_root"
 
-// OVMF firmwares.
-type ovmfFirmware struct {
+// EFI firmwares.
```

Comment:
The rework here looks good, but I think we should make a new `internal/server/instance/drivers/edk2` package which would contain those definitions and possibly some functions to access it.

That would then make it easy to consume from the apparmor package too so we don't need to keep duplicating logic between the two.

Reply:
Thanks for the feedback. I'll give it a go and push some new commits.

Reply:
Thanks!